database/migrations/2024_08_25_033712_create_categories_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('categories', function (Blueprint $table) {
+    
+        $table->id();
+        $table->string('name')->unique();
+        $table->text('description')->nullable();
```

Comment:
i don't think there is a description for a category, it is just a name and you may add an image

Reply:
do I add another table for the image?

Reply:
at the same table insert an image for each category as displayed in the website